src/test_util.rs

```diff
@@ -61,6 +61,40 @@ macro_rules! assert_lint_err {
       $($src: $test,)*
     }
   };
+
+  (
+    $rule: ty,
+    $message: expr,
+    $hint: expr,
+    filename: $filename:literal,
+    $($src:literal : $test:tt),+
+    $(,)?
+  ) => {
+      $(
+      let errors = parse_err_test!($message, $hint, $test);
+      let tester = $crate::test_util::LintErrTester::<$rule>::new(
+        $src,
+        errors,
+        $filename,
+      );
+      tester.run();
+    )*
```

Comment:
```suggestion
    $(
      let errors = parse_err_test!($message, $hint, $test);
      let tester = $crate::test_util::LintErrTester::<$rule>::new(
        $src,
        errors,
        $filename,
      );
      tester.run();
    )*
```

This should look more beautiful 👍 (it is bad that rustfmt cannot do its job on macros)